SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationButton.java

```diff
@@ -39,10 +41,20 @@ public ConversationButton(Context context, ButtonControl model, int defStyle) {
         setEllipsize(TextUtils.TruncateAt.END);
     }
 
-    private void initBorderRadius() {
-        if(model.getBorderRadius() != null) {
-            borderRadius = Float.parseFloat(model.getBorderRadius());
-            // TODO convert from percent to correct units SWRVE-11668
+    private void initBorderRadius(Context context) {
+        if (model.getBorderRadius() != null) {
```

Comment:
It is already wrapped in a try/catch in ConversationFragment:
https://github.com/Swrve/swrve-android-sdk/blob/release-4_3/SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationFragment.java#L146-L157

If an exception occurs then the conversation is not shown, which we could argue is better than giving a default.